confignode/src/main/java/org/apache/iotdb/confignode/persistence/AuthorInfo.java

```diff
@@ -77,7 +78,37 @@ public TSStatus login(String username, String password) {
     return tsStatus;
   }
 
-  public TSStatus authorNonQuery(AuthorReq authorReq) throws AuthException {
+  public TSStatus checkUserPrivileges(String username, List<String> paths, int permission) {
```

Comment:
remove to PermissionManager,  PermissionManager call AuthorInfo

Reply:
fix